templates/ruleset-scrolling-example/.template.config/template.json

```
@@ -5,12 +5,12 @@
         "Console"
     ],
     "name": "osu! ruleset (scrolling pippidon example)",
-    "identity": "ppy.osu.Game.Templates.Rulesets.Pippidon",
+    "identity": "ppy.osu.Game.Templates.Rulesets.Pippidon.Scrolling",
```

why not `Rulesets.Scrolling.Xxx` as an identity for any scrolling ruleset?
```suggestion
    "identity": "ppy.osu.Game.Templates.Rulesets.Scrolling.Pippidon",
```

---

Fine by me, but it's not like this is used for anything or is shown to anyone.